Gemfile

```diff
@@ -20,4 +20,9 @@ gem 'puppet-blacksmith', '>= 6.0.0', {"groups"=>["development"]}
 gem 'voxpupuli-acceptance', '~> 1.0', {"groups"=>["system_tests"]}
 gem 'puppetlabs_spec_helper', {"groups"=>["system_tests"]}
 
+# EL9 support
+gem 'beaker', github: 'ekohl/beaker', branch: 'EL9-support'
+gem 'beaker-docker', github: 'ekohl/beaker-docker', branch: 'el9'
+gem 'beaker-hostgenerator', github: 'ekohl/beaker-hostgenerator', branch: 'centos9'
```

Comment:
Love the consistency in the branch names 🤣

Reply:
While writing this I **knew** you'd notice :P